File: app/src/main/java/de/tum/in/tumcampusapp/models/GCMChat.java
package de.tum.in.tumcampusapp.models;


import java.io.Serializable;

/**
 * Used for parsing the GCM json payload for the 'chat' type
 */
public class GCMChat implements Serializable {
    private static final long serialVersionUID = -3920974316634829667L;
    public int room;
    public int member;
    public int message;
}

File: app/src/main/java/de/tum/in/tumcampusapp/component/ui/chat/GCMChat.kt
package de.tum.`in`.tumcampusapp.component.ui.chat

import java.io.Serializable

/**
 * Used for parsing the GCM json payload for the 'chat' type
 */
data class GCMChat(var room: Int = 0,
                   var member: Int = 0,
                   var message: Int = 0) : Serializable {
    companion object {
        private const val serialVersionUID = -3920974316634829667L
    }
}
